.changeset/proud-starfishes-walk.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-permission-backend-module-policy-allow-all': patch
+'@backstage/plugin-permission-backend': patch
+---
+
+Moved permissionModuleAllowAllPolicy to @backstage/plugin-permission-backend-module-policy-allow-all
```

Comment:
```suggestion
Moved `permissionModuleAllowAllPolicy` to `@backstage/plugin-permission-backend-module-policy-allow-all`
```

Reply:
Could also consider making this two changesets with different wording, but that's very much a nit